<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabExplorer</class>
 <widget class="QWidget" name="TabExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>资源浏览</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="topControlWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="windowTitle">
      <string>资源浏览</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineAddress">
        <property name="placeholderText">
         <string>请输入目录路径，例如：D:/data/ai_assets</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEnter">
        <property name="text">
         <string>进入</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="treeView"/>
     <widget class="QWidget" name="rightPanel">
      <layout class="QVBoxLayout" name="rightPanelLayout">
       <item>
        <widget class="QWidget" name="rightTopPanel" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="btnReference">
               <property name="text">
                <string>参考图</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnVideo">
               <property name="text">
                <string>视频</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSequenceFrames">
               <property name="text">
                <string>序列帧</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSpine">
               <property name="text">
                <string>spine</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableMeta">
         <column>
          <property name="text">
           <string>键</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>值</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
